src/languages/getNodeMatcher.ts

```diff
@@ -65,13 +66,20 @@ const languageMatchers: Record<
   markdown,
   php,
   python,
-  ruby, 
+  ruby: mergeMatchers(ruby, "ruby"),
   scala,
   typescript,
   typescriptreact: typescript,
   xml: html,
 };
 
+function mergeMatchers(
+  regexMatcher: Record<ScopeType, NodeMatcher>,
+  languageName: string
+): Record<ScopeType, NodeMatcher> {
+  return Object.assign(regexMatcher, queryBasedSpecification(languageName));
```

Comment:
It's worth checking for situations where we've specified the matcher both in the old and new matcher specifications and throwing an error, because otherwise we might override one without remembering to remove the old matcher